src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -259,9 +259,9 @@ private IReadOnlyCollection<WindowScanOutput> ScanAsyncWithProvisionForBuildAgen
             {
                 return scanner.ScanAsync(null, CancellationToken.None).Result.WindowScanOutputs;
             }
-            catch (AxeWindowsAutomationException e)
+            catch (AxeWindowsAutomationException)
```

Comment:
Maybe this should be all Exception types?
```suggestion
            catch (Exception)
```